app/assets/stylesheets/app/reservations.scss

```diff
@@ -10,6 +10,10 @@
   color: red;
 }
 
+.partially-available-warning {
+  color: #faa732;
```

Comment:
Is there a variable we can use from bootstrap for this value? Or should we wrap it in a SASS variable?

Reply:
Good call — I’ve switched this to `$orange`.